### DISCUSSION TITLE
Whats the difference between useBuiltinRegoPolicies and policiesBundle

### DISCUSSION THREAD

**Discussion by PG2000** [aquasecurity/trivy-operator#1994] [Help & Support] [answered]
Hello,

i'm wondering what the difference is between [useBuiltinRegoPolicies](https://github.com/aquasecurity/trivy-operator/blob/v0.19.4/deploy/helm/values.yaml#L493) and [policiesBundle](https://github.com/aquasecurity/trivy-operator/blob/v0.19.4/deploy/helm/values.yaml#L614C1-L614C15)

Are they mutual exclusive?

Where can i find the builtinRegoPolicies?

Could not find an answer in the docs so maybe somebody can help me with this?

**Comment by chen-keinan** [comment]
@PG2000 are the same thing, but you choose to skip `useBuiltinRegoPolicies` by setting it to `false` and then you can add your own policies to policies cm.

**Reply by PG2000** [reply]
ok this makes sense. But why do we need the [policiesBundle](https://github.com/aquasecurity/trivy-operator/blob/v0.19.4/deploy/helm/values.yaml#L614C1-L614C15)

**Reply by chen-keinan** [reply]
once you choose to use built-in policies (policies bundle) then trivy-operator will download the policies package from [public registry](https://github.com/aquasecurity/trivy-checks/pkgs/container/trivy-policies) unless you want to download the policies package on a different registry (private or else in-case of air-gapped env.) so it give you the possibility to configure it.

**Reply by PG2000** [reply]
ok i misunderstood this.
I thought builtin meant that they were already provided with the operator and nothing must be downloaded.

**Reply by chen-keinan** [reply, marked as answer]
it use to be like this in the past however now policies are downloaded as misconfig data same as vuln to de-couple it from functionality
